charts/orchestrator/templates/_helpers.tpl

```diff
@@ -120,4 +120,21 @@
         {{- end -}}
         {{- .argoCDNamespace -}}
     {{- end -}}
+{{- end -}}
+
+{{- define "get-cluster-version" -}}
+  {{- $v := "" }}
+  {{- $version :=(lookup "config.openshift.io/v1" "ClusterVersion" "" "version") }}
+  {{- range $version.status.history }}
+    {{- if eq .state "Completed" }}
+      {{- $v = (semver .version) }}
+    {{- end }}
+  {{- end }}
+
+  {{- $validMinors := list "13" "14" "15" "16" -}}
```

Comment:
Alternative way to lazily generate this list without `Major` values:
https://helm.sh/docs/chart_template_guide/function_list/#untilstep
```
{{{- list:= (untilStep 13 16 1) -}}
```